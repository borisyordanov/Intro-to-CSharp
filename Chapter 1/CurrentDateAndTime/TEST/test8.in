20
9
15
12
0
0
