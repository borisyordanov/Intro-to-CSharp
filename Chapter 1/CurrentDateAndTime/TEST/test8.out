9/15/0020 12:00:00 PM
